### PR TITLE
Link climate data

### PR DESCRIPTION
**Pull request by jinshijian**
Link climate data

### CODE REVIEW

**Comment by MaxFrissell** [comment]
I have github ignoring the Data folder, so this IGBP_Koppen_MODIS.csv file that you've added to your copy of the repo isn't in the larger repository. I'm going to make a change to the .gitignore so that it isn't ignoring this folder. Then, you can add your data to this branch and I'll merge it.

**Comment by MaxFrissell** [comment]
Alternatively, you can just send me that file and I can download it myself. Let me know what you think is best.

File: R/Max_Jinshi_Meeting.Rmd [hunk 1]
```diff
@@ -33,6 +33,7 @@ library(patchwork)
 
 ```{r data processing, echo = FALSE}
 srdbv5 <- read.csv(here::here("Data", "srdb-data.csv"))
+IGBP_Koppen_MODIS <- read.csv(here::here("Data", "IGBP_Koppen_MODIS.csv"))
```
Comment (bpbond) [inline R/Max_Jinshi_Meeting.Rmd:36]: Cool. I definitely need to use the `here` package.

File: R/Max_Jinshi_Meeting.Rmd [hunk 2]
```diff
@@ -87,4 +89,21 @@ srdbv5 %>%
...
-```
\ No newline at end of file
+```
+
+
+```{r link Climate and vegetation}
+# Need to update latitude and longitude
+# Change Latitude and Longitude to 0.5*0.5 resolution
+srdbv5 %>% 
+  mutate(Latitude2 = round(Latitude*2)/2+0.25, Longitude2 = round(Longitude*2)/2+0.25) ->
```
Comment (bpbond) [inline R/Max_Jinshi_Meeting.Rmd:99]: A more descriptive name here might be useful, e.g. `Latitude_5deg`

File: R/Max_Jinshi_Meeting.Rmd (hunk 1 above)
Comment (MaxFrissell) [inline R/Max_Jinshi_Meeting.Rmd:36]: Can I get a copy of this data? I don't know where to find it

**Comment by MaxFrissell** [comment]
Oops, I see that you emailed me this data Jinshi, thanks

**Pull request merged**